scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -717,8 +717,13 @@ def add_parse_arguments(parser = None):
         "--verbose",
         action="count",
         default=0,
-        help="Emit debugging information, call multiple times to increase "
-             "verbosity.")
+        help="Call multiple times to increase verbosity.")
+
+    parser.add_argument(
+        "-d",
+        "--debug",
+        action="store_true",
+        help="Emit debugging information.")
```

Comment:
maybe better would be to add --log-level option and pass it to setup_logging
```
    parser.add_argument('-ll', '--log-level', type=str.upper, default='INFO',
                        choices=['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'])
```

Reply:
It surely would be more extensible. The name could cause confusion given that there are two log handlers with separate log levels (File handler is always at DEBUG), but I think that any confusion could be solved by explaining it in Help section.

Reply:
IMO logging to file always as DEBUG is good, however you can add more arguments for logging functionality.
Look into pytest help, where you have: log-level, log-format, log-cli-level, log-file, log-file-level and so on ...
But then logging functionality should be in separate group in Twister help (now too much of options is in main section).